types/jasmine/jasmine-tests.ts

```diff
@@ -179,6 +179,11 @@ describe("Included matchers:", () => {
         expect(foo).toThrowError(TypeError, "foo bar baz");
     });
 
+    it("The 'toBeInstanceOf' matcher is for testing an object is an instance of the supplied class/constructor", () => {
```

Comment:
Please update also `jasmine-tests.ts` under the `ts3.1` folder and apply _exactly_ same change to keep files synchronized.